### ISSUE TITLE
[MCHANGES-112] Add more than one link template

### ISSUE THREAD

**Issue by jira-importer** [apache/maven-changes-plugin#419]
**[Olivier Lamy](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=olamy)** opened **[MCHANGES-112](https://issues.apache.org/jira/browse/MCHANGES-112?redirect=false)** and commented

In my company there are two bug tracking systems :
* functionnal team use HP Qc
* dev team use jira

The issueLinkTemplate can only be unique.
I'd like to have more than one.

---

**Affects:** 2.0-beta-3

1 votes, 1 watchers

**Comment by jira-importer** [comment]
**[Olivier Lamy](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=olamy)** commented

I'd like to have something like in the mojo configuration :

```xml
<configuration>
  <issueLinkTemplatePerSystem>
    <jira>http://wwww.key=%ISSUE%</jira>
    <qc>an other url</qc>
  </issueLinkTemplatePerSystem>
</configuration>
```

And to add a new attribute system in the element action in the changes.xml

```xml
<action dev="toto" type="fix" issue="1243434" system="qc">
  Enable retrieving component-specific issues.
</action>
```

To preserve backward compatibility, the current issueLinkTemplate will be added in the map issueLinkTemplatePerSystem with the key default .
This one will be used with action without system attribute.
WDYT ?

**Comment by jira-importer** [comment]
**[Olivier Lamy](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=olamy)** commented

fix in rev 678607 (snapshot deployed)
